test/threadstest.c

```diff
@@ -741,6 +745,142 @@ static int test_bio_dgram_pair(void)
 }
 #endif
 
+static int test_thread_reported_flags(void)
+{
+    int flags = OSSL_get_thread_support_flags();
```

Comment:
This is declare as returning `uint32_t`, but here we just use `int` which looks odd.